requirements/dev.txt

```diff
@@ -6,7 +6,7 @@ django-pipeline==3.0.0
 Django>=3.2,<4.3
 markdown
 django-localflavor==4.0
-setuptools==69.2.0
+setuptools>=69.2.0
```

Comment:
I've decided not to change anything big here for the moment and just carry on duplicating, but what's the thought process on doubling up on specifying the requirements here? For a package, I'd tend to

- put everything in `pyproject.toml`
- put your dev and ci dependencies in optional dependency groups (example of this in a PEP621 `pyproject.toml`: https://github.com/chris48s/pip-abandoned/blob/edd8537e27c6afa382d90727f482bf51f5942192/pyproject.toml#L23-L39 )
- if you want to install those packages, do `pip install -e .[dev]` or `pip install -e .[ci]` or whatever

Reply:
This seems like a reasonable approach. I think we need to figure out best practice with `pyproject.toml` files and libraries in general. It's not something I've kept up with.

Reply:
done in https://github.com/DemocracyClub/dc_django_utils/pull/88/commits/3c224a4be4dd8b08b9c6cd5d3d80319ec1015cf5 and https://github.com/DemocracyClub/dc_django_utils/pull/88/commits/69b183a5ca709ff3dda4f475866ccffe5b3a678e